effect/_intents.py

```diff
@@ -50,10 +46,23 @@ def parallel(effects):
     their results, in the same order as the input to this function.
 
     :param effects: Effects which should be performed in parallel.
+    :return: An Effect that results in a list of results, or which fails with
+        a :obj:`FirstError`.
     """
     return Effect(ParallelEffects(list(effects)))
 
 
+@attributes(['exception', 'index'])
```

Comment:
My thinking was that there was no reason to maintain the traceback information for the wrapped error since we associate that traceback with the FirstError. Maybe that's not a good way to do it? Maybe we should actually have two different tracebacks?

I have begin experimenting with a new representation for errors in the [error-obj](https://github.com/radix/effect/tree/error-obj) branch.